samza-azure/src/main/java/org/apache/samza/coordinator/AzureJobCoordinator.java

```diff
@@ -298,8 +298,9 @@ private SchedulerStateChangeListener createLeaderLivenessListener() {
   private SystemStreamPartitionGrouper getSystemStreamPartitionGrouper() {
     JobConfig jobConfig = new JobConfig(config);
     String factoryString = jobConfig.getSystemStreamPartitionGrouperFactory();
-    SystemStreamPartitionGrouper grouper = Util.getObj(factoryString, SystemStreamPartitionGrouperFactory.class)
-        .getSystemStreamPartitionGrouper(jobConfig);
+    SystemStreamPartitionGrouper grouper =
```

Comment:
The wiring for the actual plugin classloader will come in a separate PR. This PR is just setting up for that follow-up. For now, this will just continue to use the "current" classloader.

Reply:
Yeah, doing the wiring later is fine. My question is more about the naming for readability. For someone looking at these classes in isolation (i.e. outside these PRs), would it be cleaner to call these fwk / plugin classloaders explicitly? 

If I understand your point correctly, what you're saying is that once the wiring is done they will not need to know since it will just work with the following rule: if a classloader param / field is available, use it, else use your own classloader. Is that correct?

Reply:
Yep, that understanding is correct.